app/config.py

```
@@ -0,0 +1,38 @@
+from pydantic_settings import BaseSettings
+from typing import List
```

Ruff (0.14.10)

`typing.List` is deprecated, use `list` instead

(UP035)